packages/router/router.coffee

```diff
@@ -39,3 +39,22 @@ FlowRouter.route '/organizations/:_id',
     FlowLayout.render 'layout',
       main: 'organizationDetail'
       params: {"organizationId": params._id}
+
+FlowRouter.route '/myDatasets',
```

Comment:
I like the idea of having a "My Datasets" page. Not so much for space reasons, more for giving the current user easy access to their data.